.gitignore

```diff
@@ -1 +1,2 @@
 node_modules
+package-lock.json
```

Comment:
I'd ask that this be removed. not something we want to ignore in the repo. but please do refrain from adding package-lock.json in the PR